src/App.tsx

```diff
@@ -186,11 +187,27 @@ export default function App() {
 			<table>
 				<tr>
 					<td><span className={"timeline-cell label"}>Frame</span></td>
-					<td><input className={"timeline-cell"} type={"text"} value={"0"} /></td>
-					<td><input className={"timeline-cell"} type={"number"} value={frame} min={"0"} onChange={(e) => 
-						{if (parseInt(e.target.value) < markerFileData.frames.length && parseInt(e.target.value) >= 0) {setFrame(parseInt(e.target.value)); togglePlaying;}}} /></td>
-					<td><input className={"timeline-cell"} type={"number"} value={frameEnd} onChange={(e) => 
-						{if (parseInt(e.target.value) >= 0) setEnd(parseInt(e.target.value)); }} /></td>
+					<td><input className={"timeline-cell"} type={"number"} value={frameStart} min={"0"} max={markerFileData.frames.length} onChange={(e) => {
+							const inputInt = parseInt(e.target.value);
+							if (inputInt < markerFileData.frames.length && inputInt >= 0)
+							{
+								setStart(inputInt)
```

Comment:
The value set by setStart isn't getting used in the animationLoop and togglePlaying callbacks, so changes to the start of the loop aren't taking effect. This can be fixed by changing the return value on line 110 to frameStart and calling setFrame(frameStart) on line 134.